W-12976926 amazon s3 6.2 (#2293)
mulesoft/docs-connectors: amazon-s3/6.2/modules/ROOT/pages/amazon-s3-connector-reference.adoc

[[Config_Connection]]
=== Connection (S3) Connection Type

Uses the S3 connection type, which covers both the Basic and Role connections.

[%header,cols="20s,25a,30a,15a,10a"]
|===
| Name | Type | Description | Default Value | Required
| Access Key a| String |  Access key provided by Amazon. |  | x
| Secret Key a| String |  Secret key provided by Amazon. |  | x
| Region Endpoint a| String |  Topic region endpoint. |  `us-east-1` |
| Connection Timeout a| Number |  How long the connector waits before timing out when establishing a connection to the remote service. A value of `0` means infinity and is not recommended. |  `50` |
| Connection Timeout Unit a| Enumeration, one of:

** NANOSECONDS
** MICROSECONDS
** MILLISECONDS
** SECONDS
** MINUTES
** HOURS
** DAYS | Time unit for the *Connection Timeout* field. |  `SECONDS` |
| Response Timeout a| Number | How long Mule waits for a response to complete before the request times out. |  `30` |
| Response Timeout Unit a| Enumeration, one of:

** NANOSECONDS
** MICROSECONDS
** MILLISECONDS
** SECONDS
** MINUTES
** HOURS
** DAYS |  Time unit for the *Response Timeout* field. |  `SECONDS` |
| Role a| <<Role>> |  Role configuration. |  |

> | Try Default AWSCredentials Provider Chain a| Boolean |  Set this field to `true` to obtain credentials from the AWS environment. For more information, refer to https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[Provide Temporary Credentials to the SDK

Author: ```suggestion
| Try Default AWSCredentials Provider Chain a| Boolean |  If `true`, obtains credentials from the AWS environment. For more information, refer to https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[Provide Temporary Credentials to the SDK
```

** HOURS
** DAYS |  Time unit for the *Response Timeout* field. |  `SECONDS` |
| Role a| <<Role>> |  Role configuration. |  |

> | Try Default AWSCredentials Provider Chain a| Boolean |  Set this field to `true` to obtain credentials from the AWS environment. For more information, refer to https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[Provide Temporary Credentials to the SDK

Reviewer: An optional suggestion to tighten the wording: 

If `true`, obtains credentials from the AWS environment
